### PR TITLE
Make Execute() wait for initialization of status gRPC server before use

### LINKED ISSUE #828: Cluster panic
Some time after start, all nodes of my cluster (two servers and two agents) crash with the same error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x160 pc=0x948a8a]

goroutine 623378 [running]:
google.golang.org/grpc.(*Server).Stop(0x0)
        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:1482 +0x4a
github.com/distribworks/dkron/v3/plugin.(*ExecutorClient).Execute(0xc00091e6a0, 0xc00b231640, 0x357c300, 0xc000e27820, 0xc0011bc980, 0x25bb801, 0xc0044c6150)
        /home/runner/work/dkron/dkron/plugin/executor.go:65 +0x190
github.com/distribworks/dkron/v3/dkron.(*AgentServer).AgentRun(0xc0001295b8, 0xc0044c6060, 0x35e2620, 0xc003374e20, 0x0, 0x0)
        /home/runner/work/dkron/dkron/dkron/grpc_agent.go:83 +0x6f4
github.com/distribworks/dkron/v3/plugin/types._Agent_AgentRun_Handler(0x2561d80, 0xc0001295b8, 0x35dcda0, 0xc0002186c0, 0x4ba5ae0, 0xc000e2a200)
        /home/runner/work/dkron/dkron/plugin/types/dkron.pb.go:1862 +0x109
google.golang.org/grpc.(*Server).processStreamingRPC(0xc000a12b60, 0x35ea7e0, 0xc0019b4300, 0xc000e2a200, 0xc000b56150, 0x4b45e20, 0x0, 0x0, 0x0)
        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:1329 +0xcbc
google.golang.org/grpc.(*Server).handleStream(0xc000a12b60, 0x35ea7e0, 0xc0019b4300, 0xc000e2a200, 0x0)
        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:1409 +0xc64
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc001ab37e0, 0xc000a12b60, 0x35ea7e0, 0xc0019b4300, 0xc000e2a200)
        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:746 +0xa1
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /home/runner/go/pkg/mod/google.golang.org/grpc@v1.29.1/server.go:744 +0xa1
runtime: note: your Linux kernel may be buggy
runtime: note: see https://golang.org/wiki/LinuxKernelSignalVectorBug
runtime: note: mlock workaround for kernel bug failed with errno 12
```

### Environment

- Dkron 3.0.5
- Docker (servers: [Dockerfile](https://github.com/alexanderfefelov/docker-backpack/blob/master/workload-automation/dkron/server/Dockerfile), [run script](https://github.com/alexanderfefelov/docker-backpack/blob/master/workload-automation/dkron/server/functions.sh), agents: [Dockerfile](https://github.com/alexanderfefelov/docker-backpack/blob/master/workload-automation/dkron/agent/Dockerfile), [run script](https://github.com/alexanderfefelov/docker-backpack/blob/master/workload-automation/dkron/agent/functions.sh))
- `uname -a`:

        Linux dkron-server-1.backpack.test 5.4.0-48-generic #52-Ubuntu SMP Thu Sep 10 10:58:49 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

### PR DESCRIPTION
**Pull request by yvanoers**
Attempt to fix #828 .
I have no decent way to test whether this fixes the problem, so would be great if someone having issues could test this.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
           DOCKER_REPO: ${{ secrets.DOCKER_REPO }}
         run: |
           docker buildx build \
-            --platform linux/amd64,linux/arm64,linux/arm/v7 \
+            --platform linux/amd64 \
```
Comment (victorcoder) [inline .github/workflows/release.yml:56]: Why remove arm? for the testing?

Comment (yvanoers) [reply .github/workflows/release.yml:56]: The release build failed on arm64 without returning a clear error message. I didn't want to spend a lot of time debugging that, so I just removed arm.
I will revert the change eventually.

Comment (yvanoers) [reply .github/workflows/release.yml]: This has been removed from the branch.

**Review by victorcoder — APPROVED** [review]
LGTM

**Pull request merged**